src/Network/Network/Models/AzureFirewall/PSAzureFirewall.cs

```diff
@@ -386,6 +386,24 @@ public void RemovePublicIpAddress(PSPublicIpAddress publicIpAddress)
             }
         }
 
+        public void AddIpAddressesForByopipHubFirewall(PSPublicIpAddress[] publicIpAddresses)
+        {
+            this.IpConfigurations = new List<PSAzureFirewallIpConfiguration>();
+
+            if (publicIpAddresses != null && publicIpAddresses.Count() > 0)
+            {
+                for (var i = 0; i < publicIpAddresses.Count(); i++)
```

Comment:
Count or Length is preferable over Count() as this is an extension method from Linq which may cause performance overhead.